lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -177,7 +177,7 @@ export abstract class CacheManager implements ICacheManager {
      * retrieve a credential - accessToken, idToken or refreshToken; given the cache key
      * @param key
      */
-    getCredential(key: string): CredentialEntity | null {
+    getCredential(key: string): CredentialEntity {
```

Comment:
Why were these changes made? The interface should be updated to include the null return instead

Reply:
An object can be null, why are we specifically making null as an acceptable type? Either way is fine for me, the changes were made because AzureSDK is seeing these issues in their repo:
    
_../../../common/temp/node_modules/.pnpm/@azure/msal-common@1.3.0/node_modules/@azure/msal-common/dist/src/cache/CacheManager.d.ts(78,5): error TS2416: Property 'getCredential' in type 'CacheManager' is not assignable to the same property in base type 'ICacheManager'. 
 Type '(key: string) => CredentialEntity | null' is not assignable to type '(key: string) => CredentialEntity'. 
 Type 'CredentialEntity | null' is not assignable to type 'CredentialEntity'. 
 Type 'null' is not assignable to type 'CredentialEntity'._

Reply:
Done. Made them uniform and kept the `null`. Lets revisit this when we enable `strict null checks` in the library.